tests/subsys/zigbee/osif/nvram/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/dts.overlay")
+  set(DTC_OVERLAY_FILE "${CMAKE_CURRENT_SOURCE_DIR}/dts.overlay")
+endif()
+
+include($ENV{ZEPHYR_BASE}/../nrf/cmake/boilerplate.cmake)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(zigbee_osif_nvram_test)
+
+add_definitions(-Wno-packed-bitfield-compat)
+zephyr_compile_definitions(CONFIG_ZBOSS_OSIF_LOG_LEVEL=3)
```

Comment:
this looks like something that should be specified in a conf file.

Reply:
This test only tests a single part of zigbee subsys (the nvram component) and the zigbee subsystem isn't used. `CONFIG_ZBOSS_OSIF_LOG_LEVEL` is needed for this test but is dependent on zigbee and therefore unavailable in proj.conf.